exercises/concept/interest-is-interesting/.docs/introduction.md

```diff
@@ -41,3 +41,10 @@ do
     x = x - 2;
 } while (x > 10)
 ```
+
+## Simple Interest & Compounding Interest
+
+When applying simple interest to a principle balance, the balance is multipled by the interest rate and the product of the two is the interest amount.
```

Comment:
Principal, not principle (a common mistake)

Reply:
oh me and spelling, thanks for pointing this out